### ISSUE TITLE
Setting calculation memory

### ISSUE THREAD

**Issue by LeGmask** [Omega-Numworks/Omega#214]
#### What I want to see in the next version of Omega
---------------------------------------------------------------
I had an idea, because sometimes when I make big calculations and I haven't clear memory of calculation; I take a full working memory, therefore I do not know if it would be possible to create a limit of x calculation in the memory you can choose in a frame? But should help some people in my class

**Comment by LeGmask** [comment]
okay little update, the app just keep 25 calculation in memory

**Comment by Moddingear** [comment]
I agree, I have been running into the same issue, but it only happens in omega.
I think more memory should be given to calculations

**Comment by quentinguidee** [comment]
@Moddingear In Epsilon 13, the ram will be better managed :) So, when Omega will be updated with E13, everything will be fixed :)

**Comment by LeGmask** [comment]
Yeah but the calculation memory in setting, is a good idea !

**Comment by LeGmask** [comment]
> 
> 
> @Moddingear In Epsilon 13, the ram will be better managed :) So, when Omega will be updated with E13, everything will be fixed :)

Hmmm actually it still generate the issue with e13
